kiva/fonttools/_scan_parse.py

```diff
@@ -58,14 +58,16 @@ class FontEntry(object):
     """ A class for storing Font properties. It is used when populating
     the font lookup dictionary.
     """
-    def __init__(self, fname="", name="", style="normal", variant="normal",
-                 weight="normal", stretch="normal", size="medium"):
+    def __init__(self, fname="", family="", style="normal", variant="normal",
+                 weight="normal", stretch="normal", size="medium",
+                 face_index=0):
         self.fname = fname
-        self.name = name
+        self.family = family
         self.style = style
         self.variant = variant
         self.weight = weight
         self.stretch = stretch
+        self.face_index = face_index
```

Comment:
This change is integrating the `face_index` pieces that we're introduced by #605 to the refactored font parsing code.